src/licensedcode/data/licenses/freetos-mit.LICENSE

```diff
@@ -0,0 +1,32 @@
+---
+key: freetos-mit
```

Comment:
@DennisClark maybe a typo here?
```suggestion
key: freertos-mit
```

Reply:
oops

Reply:
DejaCode updates: 
* deprecated freetos-mit, setting it to inactive (by design, cannot change the license key in DejaCode)
* created freertos-mit